### PR TITLE
Outline empty states

### PR DESCRIPTION
**Pull request by dr15**
Uses previously created empty states in the build outline, one for empty course and one for empty lesson.
They are added in a way that the stay in place when an item is added (basically they are siblings of the dnd component and are set as position absolute)

### CODE REVIEW

File: src/images/icons/empty-course.svg [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
+      <path stroke-linecap="round" d="M52.655 10.388h-23.26c-3.05 0-4.575 1.115-4.575 3.343 0 2.675 1.525 4.012 4.575 4.012H51.13m-21.915-3.678h21.19"/>
+    </g>
+    <g fill="#8D8D8D">
+      <path d="M52.655 9.653a.735.735 0 01.132 1.46l-.132.01h-23.26c-2.687 0-3.843.845-3.843 2.608 0 .432.064.822.182 1.17l.061.165a3.38 3.38 0 002.801 1.854l.214.006h22.38c1.161 0 2.112.904 2.192 2.05l.006.157v22.145c0 1.165-.9 2.12-2.04 2.2l-.158.006H29.215c-2.76 0-5.01-2.188-5.123-4.93l-.005-.218v-23.81c0-.063.007-.122.019-.178a9.636 9.636 0 01-.019-.617c0-2.601 1.766-3.981 4.96-4.073l.348-.005h23.26zm-27.103 7.605v21.078a3.671 3.671 0 003.468 3.672l.195.005H51.19a.734.734 0 00.726-.636l.007-.1V19.134a.736.736 0 00-.401-.656h-17.18v13.975c0 .71-.887.991-1.302.464l-.058-.086-1.57-2.627-1.57 2.627c-.357.6-1.24.398-1.349-.244l-.01-.134V18.386a4.817 4.817 0 01-2.333-.788 4.615 4.615 0 01-.598-.34zm7.325 1.22h-2.93v11.318l.837-1.4a.731.731 0 011.177-.109l.08.11.836 1.4V18.478zm17.528-5.148a.735.735 0 01.133 1.459l-.133.012h-21.19a.735.735 0 01-.132-1.46l.132-.01h21.19z"/>
     </g>
   </g>
-</svg>
+</svg>
```
Comment (nunibaranes) [inline src/images/icons/empty-course.svg:13]: No newline at the end of the file

File: src/images/icons/empty-lesson.svg [hunk 2]
```diff
@@ -1,21 +1,6 @@
...
   </g>
-</svg>
+</svg>
```
Comment (nunibaranes) [inline src/images/icons/empty-lesson.svg:6]: No newline at the end of the file

Comment (dr15) [reply src/images/icons/empty-lesson.svg:6]: Done

File: src/images/icons/empty-course.svg (hunk 1 above)
Comment (dr15) [reply src/images/icons/empty-course.svg:13]: Done

**Pull request merged**